src/components/dapp-staking/stake-manage/ModalSelectFunds.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <AstarModal
+  <astar-modal
```

Comment:
if not using astar-ui library, please not using like astar-[componentName] . the format already used in ui library as default, it might be conflicts. if you want to implement the common component, feel free to contributing in ui library instead. @impelcrypto @bobo-k2

Reply:
@sirius651 Ok. As for AstarModal, this is a wrapper component for astar-simple-modal and it cannot be defined in Astar-UI because I want to use `Teleport`. [Ref](https://github.com/AstarNetwork/astar-apps/blob/main/src/components/common/AstarModal.vue) I'll change this component name.

Reply:
@sirius651 I've fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/533/commits/939efb7e595d8c1cc2163a1bdb1f2ff56078a92e) commit.